build.gradle

```diff
@@ -17,7 +17,7 @@ sourceCompatibility = JavaVersion.VERSION_1_8
 targetCompatibility = JavaVersion.VERSION_1_8
 
 group 'org.zaproxy'
-version '0.15.0'
+version '0.16.0-SNAPSHOT'
```

Comment:
Why snapshot?

Reply:
It indicates that's a dev version (i.e. not released), Zest follows this pattern (like `zaproxy` repo does).